frontend/apps/core-app/src/screens/RecipientListScreen/RecipientListScreen.component.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Button, Text, VStack } from 'native-base';
+import { NavigationProp, RouteProp } from '@react-navigation/native';
+
+import { RootParamList } from '../../navigators/types';
+
+import * as Styles from './RecipientListScreen.styles';
+
+type Props = {
+  route: RouteProp<RootParamList, 'RecipientList'>;
+  navigation: NavigationProp<RootParamList>;
+};
+
+export const RecipientListScreenComponent: React.FC<Props> = ({
+  route,
+  navigation,
+}) => {
+  return (
+    <Styles.Container>
+      <Text variant="display">{route.name}</Text>
+      <VStack space={2} width="sm">
```

Comment:
Here the spacing will use the native base spacing, not nrc spacing. To use nrc spacing this would need to be `nrc_x`. We should discuss this again outside of this pr.

Reply:
generally yeah, but I didn't see the need to pretend like this is even close to a real design ^^

Reply:
Yeah, not important for this instance. But I think it is important for defining the spacing in the design system.